vars.sample

```diff
@@ -50,10 +50,20 @@ ORACLE_TABLESPACES_DIR=/home/oracle/app/oracle/oradata
 # }}}
 
 # Only needed for ETL
+ETL_SCRIPTS=/path/to/transmart-data/samples/postgres
+export PATH=$ETL_SCRIPTS:$PATH
+
+# export KETTLE_HOME=$HOME ## kettle.properties file is expected in .kettle subdirectory of KETTLE_HOME
```

Comment:
And by the way, even if you, for whatever reason don't want to launch the scripts with make, you could still, invoke make to create a up-to-date kettle home.